include/slang-rhi.h

```diff
@@ -1401,6 +1409,8 @@ struct AccelerationStructureDesc
     const void* next = nullptr;
 
     uint64_t size;
+    AccelerationStructureBuildFlags flags = AccelerationStructureBuildFlags::None;
+    AccelerationStructureMotionInfo* motionInfo = nullptr;
```

Comment:
These extra options feel wrong to me.
An acceleration structure should just be a chunk of memory. I haven't looked at why this is needed specifically, but we should try to get rid of this.